scss/_card.scss

```diff
@@ -91,6 +91,10 @@
 // Background variations
 //
 
+.card-default {
+  background-color: #fff;
```

Comment:
As odd as it may seem, wouldn't it be more consistent to go with something such as `.card-secondary` to stay consistent with button variants, class names, and user's expectations?